### ISSUE TITLE
Images

### ISSUE THREAD

**Issue by stefanoco** [themarshallproject/pony#10]
Pony sounds really useful. Any idea on if/how to include images in the markdown text? Just plain markdown sintax and image files served from some external URI?

**Comment by tommeagher** [comment]
@stefanoco Exactly. We store an image on S3 or somewhere else and use the [markdown image syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images).
